app/views/add-payment-hold.njk

```diff
@@ -0,0 +1,63 @@
+{% extends '_layout.njk' %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [{
+      text: "Home",
+      href: "/"
+    },{
+      text: "Payment Holds",
+      href: "/payment-holds"
+    }]
+  })}}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <form method="POST" action="/add-payment-hold" autocomplete="off" novalidate>
+        <input type="hidden" name="crumb" value="{{crumb}}"/>
+        <h1 class="govuk-heading-l">Add payment hold</h1>
+
+        <dl class="govuk-summary-list">
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">FRN</dt>
+            <dd class="govuk-summary-list__value">
+              {% if paymentHoldFRNs.length > 0 %}
+                <select class="govuk-select" id="frn" name="frn">
+                  {% for paymentHoldFRN in paymentHoldFRNs %}
+                    <option value="{{ paymentHoldFRN.frn }}">{{ paymentHoldFRN.frn}} </option>
+                  {% endfor %}
+                </select>
```

Comment:
Rather than a drop down, could we just have a numeric text box?

Don't worry about validation for now but just want to allow users to enter any FRN rather than restricted to a random 20.